src/mcts/stoppers/alphazero.cc

```diff
@@ -70,6 +70,12 @@ std::unique_ptr<SearchStopper> AlphazeroTimeManager::GetStopper(
       (alphazerotimepct_ / 100.0f) + *increment *
       (alphazeroincrementpct_ / 100.0f);
 
+  // If increment time is added only after a move has been made,
```

Comment:
I realized that `*increment * (alphazeroincrementpct_ / 100.0f)` can still get higher than available time.
Now the calculation looks safe. If increment is added before the move is made (e.g. in Cutechess), the code works without precautions as in bbfa544. If increment is added after, 1) `*increment` or 2) `*increment * (alphazeroincrementpct_ / 100.0f)` can be higher than available time and this is now covered with (1c21588) and this commit.